src/main/java/com/amalitech/gpuconfigurator/dto/profile/BasicInformationResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.amalitech.gpuconfigurator.dto.profile;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class BasicInformationResponse {
```

Comment:
Use a projection for this, It'll save you save some boilerplate code in your service

https://docs.spring.io/spring-data/jpa/reference/repositories/projections.html